processFlows.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "name": "demand-create",
+    "name": "demand-propose",
```

Comment:
?

Reply:
the token has `state: proposed`, was going for consistency. Could go other way and make `state: created`

Reply:
I think we should have state created, but let me do it in my upcoming PR